sktime/datatypes/_dtypekind.py

```diff
@@ -0,0 +1,69 @@
+from enum import IntEnum
+
+import numpy as np
+import pandas as pd
+
+
+class DtypeKind(IntEnum):
+    """
+    Integer enum for data types.
+
+    Attributes
+    ----------
+    INT : int
+        Matches to signed integer data type.
+    UINT : int
+        Matches to unsigned integer data type.
+    FLOAT : int
+        Matches to floating point data type.
+    BOOL : int
+        Matches to boolean data type.
+    STRING : int
+        Matches to string data type (UTF-8 encoded).
+    DATETIME : int
+        Matches to datetime data type.
+    CATEGORICAL : int
+        Matches to categorical data type.
+    """
+
+    INT = 0
+    UINT = 1
+    FLOAT = 2
+    BOOL = 20
+    STRING = 21  # UTF-8
+    DATETIME = 22
+    CATEGORICAL = 23
+
+
+def _simplify_dtype(col_dtypes):
```

Comment:
perhaps a different name is appropriate here?

Reply:
Hmm, maybe `dtype_to_kind` ?

Reply:
it's not a simplification because input and output do not live in the same "domain", it is a translation

Reply:
okay, any suggestions on the name then?